orm/src/test/java/org/hibernate/search/test/interceptor/Blog.java

```diff
@@ -29,6 +31,18 @@ public void setId(Integer id) {
 		this.id = id;
 	}
 
+	@Override
+	public boolean equals(Object obj) {
+		return getClass().equals( obj.getClass() )
+				&& id != null
+				&& Objects.equals( id, ((Blog) obj).id );
+	}
+
+	@Override
+	public int hashCode() {
+		return getClass().hashCode();
```

Comment:
I'm surprised the `hashCode` only use the class `hashCode`, while the `equals` involves the id?